client/src/components/Workflow/Editor/LintSection.vue

```diff
@@ -1,22 +1,31 @@
 <template>
-    <div class="lint-section">
-        <b-button
-            :aria-expanded="expanded ? 'true' : 'false'"
-            :aria-controls="collapseId"
-            :variant="variant"
-            @click="expanded = !expanded"
-        >
-            <font-awesome-icon icon="check" v-if="okay" />
-            <font-awesome-icon icon="times" v-else />
-            {{ title }}
-            <font-awesome-icon icon="angle-double-up" v-if="expanded" />
-            <font-awesome-icon icon="angle-double-down" v-else />
-        </b-button>
-        <b-collapse :id="collapseId" v-model="expanded">
-            <div class="lint-section-body">
-                <slot> </slot>
+    <div class="mb-2">
+        <div v-if="isOkay">
+            <font-awesome-icon icon="check" class="text-success" />
+            <span>{{ successMessage }}</span>
+        </div>
+        <div v-else>
+            <font-awesome-icon icon="exclamation-triangle" class="text-warning" />
+            <span>{{ warningMessage }}</span>
```

Comment:
Something that's available to all components that use the standard mount is the localize filter and the v-localize directive. You could be using {{ warningMessage | localize }} here (and on every other message).

Reply:
Added the localize option to the warningMessage.

Reply:
I notice you did it on just the one message. Maybe we should schedule another PR that goes through all the components and removes adds localization when appropriate.